docs/templates/tutorial.md

```diff
@@ -0,0 +1,133 @@
+# Your first template
+
+A common way to create a template is to begin with a starter template
+then modify it for your needs. Coder makes this easy with starter
+templates for popular development targets like Docker, Kubernetes,
+Azure, and so on. Once your template is up and running, you can edit
+it in the Coder dashboard. Coder even handles versioning for you so you
+can publish official updates or revert to previous versions.
+
+In this tutorial, you'll create your first template from the Docker
+starter template.
+
+
+## Before you start
+
+You'll need a computer or cloud computing instance with both
+[Docker](https://docs.docker.com/get-docker/) and
+[Coder](../install/index.md) installed on it.
+
+> When setting up your computer or computing instance, make sure to
+> install Docker first, then Coder.
```

Comment:
```suggestion
> When setting up your computer or computing instance, make sure to
> install Docker first, then Coder. Otherwise, you'll need to
> give the [Coder user permission to use Docker](https://github.com/coder/coder/blob/main/examples/templates/docker/README.md#infrastructure).
```

This link doesn't work yet but it will once [this branch](https://github.com/coder/coder/blob/2b867b79c379388e982aa982aee20f3e2469239a/examples/templates/docker/README.md#infrastructure) is merged.